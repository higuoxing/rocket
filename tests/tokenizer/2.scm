;; RUN: diff --color -u <(cat %s.expected) <(rsi --debug-dump-tokens %s 2>&1)
(+)
(+1)
(+ 1)
