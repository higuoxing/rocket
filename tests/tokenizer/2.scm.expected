(2:0) LPAREN: (
(2:1) IDENTIFIER: +
(2:2) RPAREN: )
(3:0) LPAREN: (
(3:1) NUMBER: +1
(3:3) RPAREN: )
(4:0) LPAREN: (
(4:1) IDENTIFIER: +
(4:3) NUMBER: 1
(4:4) RPAREN: )
(5:0) EOF: 
